boards/nxp/imx943_evk/imx943_evk_mimx94398_a55_defconfig

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# ARM Options
+CONFIG_AARCH64_IMAGE_HEADER=y
+CONFIG_ARMV8_A_NS=y
+
+# MMU Options
+CONFIG_MAX_XLAT_TABLES=24
+
+# Cache Options
+CONFIG_CACHE_MANAGEMENT=y
+CONFIG_DCACHE_LINE_SIZE_DETECT=y
+CONFIG_ICACHE_LINE_SIZE_DETECT=y
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
+CONFIG_KERNEL_DIRECT_MAP=y
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+CONFIG_CLOCK_CONTROL=y
+
+CONFIG_MBOX=y
+CONFIG_ARM_SCMI=y
+CONFIG_INTC_INIT_PRIORITY=2
+CONFIG_MBOX_INIT_PRIORITY=3
```

Comment:
why are init priorities being set in a defconfig file?

Reply:
We need to make sure the GIC interrupt controller (which is 40 by default)  to be earlier than mailbox initialization (which default init priority is also 40), and as this platform is based on SCMI, so need to make GIC and mbox initialization to be earlier than any device initialization.
is thera a better location to set these init priorities except in defconfig

Reply:
Hi, @nordicjm , I go through the document for https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html, but still not very clear the difference between board_defconfig and Kconfig.defconfg, is there any specifc rules we could follow? thanks.

Reply:
*_defconfig is for setting values specifically, so you enable a UART because that should be enabled by default. Kconfig.defconfig is for adjusting defaults that get merged with other options, so you might only want to set a default if e.g. USB is enabled, in terms of init priority you set a default for the value which means it can be overridden in other places if needed, you are not setting it to a specific value, you are changing the value it defaults to to a specific value

Reply:
Clear for me, will update it, thanks @nordicjm